srcDLL/op2ext.h

```diff
@@ -61,12 +61,12 @@ OP2EXT_API void Log(const char* message);
 
 
 // Performs a case insensitive search of loaded module names, returning true if found.
-// A console module's name is the name of the directory the console module is stored in.
+// A console module's name is the relative directory passed in via the /loadmod command.
 // An ini module name is the module's [section name] within the ini file.
 OP2EXT_API bool IsModuleLoaded(const char* moduleName);
 
 // Performs a case insensitive comparison of the loaded console module name.
-// A console module's name is the name of the relative directory the console module is stored in.
+// A console module's name is the relative directory passed in via the /loadmod command.
```

Comment:
Maybe we can reference an earlier comment or help section, rather than repeating the explanation.